src/DebugChannel.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+import path from "path";
+
+export class DebugChannel {
+  constructor(fs: any) {
+    let configFilePath = path.join(__dirname, "../src/debugger/config.json");
```

Comment:
Things in the src directory should not have hard coded paths. Rather pass these values in as a parameter.